specifyweb/frontend/js_src/lib/components/WbPlanView/__tests__/navigator.test.ts

```diff
@@ -20,396 +20,405 @@ theories(getMappingLineData, [
-          cataloger: {
-            optionLabel: 'Cataloger',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: false,
-            isRelationship: true,
-            tableName: 'Agent',
-          },
-          collectionObjectAttribute: {
-            optionLabel: 'Col Obj Attribute',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: false,
-            isRelationship: true,
-            tableName: 'CollectionObjectAttribute',
-          },
-          collectionObjectCitations: {
-            optionLabel: 'Collection Object Citations',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: false,
-            isRelationship: true,
-            tableName: 'CollectionObjectCitation',
-          },
-          determinations: {
-            optionLabel: 'Determinations',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: true,
-            isRelationship: true,
-            tableName: 'Determination',
-          },
-          dnaSequences: {
-            optionLabel: 'DNA Sequences',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: false,
-            isRelationship: true,
-            tableName: 'DNASequence',
-          },
-          collectingEvent: {
-            optionLabel: 'Field No: Locality',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: false,
-            isRelationship: true,
-            tableName: 'CollectingEvent',
-          },
-          collection: {
-            isDefault: false,
-            isEnabled: true,
-            isHidden: false,
-            isRelationship: true,
-            isRequired: false,
-            optionLabel: 'Collection',
-            tableName: 'Collection',
-          },
-          collectionObjectAttachments: {
-            isDefault: false,
-            isEnabled: true,
-            isHidden: false,
-            isRelationship: true,
-            isRequired: false,
-            optionLabel: 'Collection Object Attachments',
-            tableName: 'CollectionObjectAttachment',
-          },
-          preparations: {
-            optionLabel: 'Preparations',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: false,
-            isRelationship: true,
-            tableName: 'Preparation',
-          },
-          voucherRelationships: {
-            optionLabel: 'Voucher Relationships',
-            isEnabled: true,
-            isRequired: false,
-            isHidden: false,
-            isDefault: false,
-            isRelationship: true,
-            tableName: 'VoucherRelationship',
-          },
+        "customSelectSubtype": "simple",
```

Comment:
hard to see the diff here, especially since a bunch of quotes were added (that's why we have Prettier)
could you please summarize what changed here?